Tasks/ArchiveFiles/task.json

```diff
@@ -70,7 +70,7 @@
             "groupName": "archive",
             "visibleRule": "archiveType = tar",
             "options": {
-                "default": "gz",
```

Comment:
Is there any harm in removing "default" from the list? (I assume not, but may be worth testing how an existing definition with the value default works with this change).